### ISSUE TITLE
Server crashes after using vehicle.destroy()

### ISSUE THREAD

**Issue by mrgharabaghi** [altmp/altv-issues#1798]
### Description of the problem

```js
chat.registerCmd('car', async (player, args) => {
    try {
        while (true) {
            const newVehicle = new alt.Vehicle(
                args[0],
                ...rotationToDirection(player.pos, player.rot, 5),
                0,
                0,
                player.rot.z + 1.5
            );

            await new Promise((res) => {
                setTimeout(() => {
                    res();
                    newVehicle.destroy();
                }, 1000);
            });
        }
    } catch (error) {
        alt.log('~c~Spawn Car Error ~r~' + error);
    }
});
```

### Reproduction steps

1. Run the server
2. Open alt:V
3. Connect to the server
4. `/car vigero`
5. After a few spawns, server will crash

### Expected behaviour

Should not crash.

### Additional context

Check the `crashdumps` folder.

### Operating system

Windows 10

### Version

dev/15.0-dev13

### Reproduction tested

- [X] I confirm that I have made sure that this issue is also present on the newest dev version

**Comment by FabianTerhorst** [comment]
You are doing a none awaited endless loop server. So it freezes the server.

**Comment by mrgharabaghi** [comment]
That `while` loop is only for testing purposes.
I used `await` in line 12.

**Comment by xxshady** [comment]
Can't reproduce on 15.0-dev15

**Comment by mrgharabaghi** [comment]
> Can't reproduce on 15.0-dev15

How many cars spawned?
@xxshady

**Comment by xxshady** [comment]
A lot of, and different ones

**Comment by mrgharabaghi** [comment]
Did you use `player.spawn(x, y, z)`?

**Comment by xxshady** [comment]
On player connect, yes

**Comment by mrgharabaghi** [comment]
Use this as the only resource on the server, please.
And wait for more than 20 times to spawn.
```toml
modules = [
    'js-module'
]

resources = [
    'bennys'
]
```
[bennys.zip](https://github.com/altmp/altv-issues/files/11117342/bennys.zip)

**Comment by xxshady** [comment]
@mrgharabaghi 2 minutes, no crash on 15.0-dev16

![изображение](https://user-images.githubusercontent.com/54737754/229085952-d179f519-632f-4918-a5c4-90a7b0413996.png)

![изображение](https://user-images.githubusercontent.com/54737754/229086126-6925dd22-f9a7-4f5a-a8fb-1d15a75660c0.png)

**Comment by mrgharabaghi** [comment]
I sent the `crashdump` file to your Discord DM.

**Comment by mrgharabaghi** [comment]
Fixed in `dev/15.0-dev20`
